CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Feature
 
+- add control panel via config.settings @ksuess https://github.com/plone/volto/issues/3426
+- Add noindex metadata tag @steffenri
+- Send extra data coming from listing block schemaEnhancer from searchBlock to the listing variation @ionlizarazu
```

Comment:
@ionlizarazu @sneridagh uh, it looks like some extra changelog entries got added here